cli/injector/grpc_disruption.go

```diff
@@ -72,7 +72,10 @@ var grpcDisruptionCmd = &cobra.Command{
 		// create injectors
 		for i, config := range configs {
 			if i == 0 {
-				injectors = append(injectors, injector.NewGRPCDisruptionInjector(spec, injector.GRPCDisruptionInjectorConfig{Config: config}))
+				injectors = append(injectors, injector.NewGRPCDisruptionInjector(spec, injector.GRPCDisruptionInjectorConfig{
+					Config:      config,
+					TimeoutInMs: 5000,
```

Comment:
Usually in go you use duration, they are easily parseable even when provided as arguments from the command line.
It would feel more natural to have here `Timeout` as a `time.Duration`.

You might want to create a const to make it more readable and explain briefly the arbitrary chosen value.